ci/common/build.sh

```diff
@@ -95,4 +95,7 @@ macos_rvm_dance() {
   rvm reload
   rvm use 2.2.5
   rvm use
+  # workaround against rvm overriding cd (revisit)
+  # https://github.com/travis-ci/travis-ci/issues/8703#ref-pullrequest-279112274
+  unset -f cd
```

Comment:
macos_rvm_dance is not actually used, see https://github.com/neovim/neovim/pull/7820#issuecomment-355844949

I don't think we call rvm directly at all. I wonder if travis's own setup has a problem (would not be surprised, they have many problems, e.g. with python).